ask.yaml

```diff
@@ -0,0 +1,20 @@
+duration: 8h
+price: 23.73 SNM/h
+
+blacklist: 0x8125721c2413d99a33e351e1f6bb4e56b6b633fd
+
+resources:
+  cpu:
+    cores: 150
+  ram:
+    size: 2gb
```

Comment:
Replace with uppercase `GB`. It's SI: https://en.wikipedia.org/wiki/Gigabyte - the Standard. Do not violate it. Ever.